examples/juniper/cptx-host/juniper.config

```diff
@@ -6,16 +6,6 @@ system {
         ssh {
             root-login allow;
         }
-        extension-service {
```

Comment:
why are you removing this config

Reply:
Having this config part of the default config causes a race where gRPC service is started before the certer interface is invoked leading to failure. This should be configured after self-signed certs are configured.